src/Misc/layoutbin/RunnerService.js

```diff
@@ -3,94 +3,136 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 var childProcess = require("child_process");
-var path = require("path")
+var path = require("path");
+const { exit } = require("process");
```

Comment:
Why do we need to import process now if we didn't use it before?

Reply:
We don't. My bad... I will fix it now